### PR TITLE
Batch update

### PR DESCRIPTION
**Pull request by EchterAgo**

### CODE REVIEW

**Comment by EchterAgo** [comment]
Updated based on response by Billfodl

https://www.reddit.com/r/btc/comments/a501i2/dear_billfodl_you_sell_a_product_for/ebl539v/

**Comment by EchterAgo** [comment]
@kenman345 this is also ready

File: _data/payments.yml [hunk 1]
```diff
@@ -204,6 +204,8 @@ websites:
   bch: false
   btc: true
   othercrypto: true
+  exceptions:
+    text: GloBee used to accept BCH, but on 2018-11-23 stopped accepting it
```
Comment (kenman345) [inline _data/payments.yml]: not certain but is that a specail character comma?

Comment (EchterAgo) [reply _data/payments.yml]: My hex editor says 0x2c, normal comma

**Pull request merged**